util/topsql/tracecpu/profile.go

```diff
@@ -15,35 +15,24 @@
 package tracecpu
```

Comment:
As now there is a package called `topsql/reporter`, then how about moving this to `topsql/collector/cpu.go`?

Reply:
You mean to create a new pkg which name is `collector`?

Reply:
`tracecpu` and `stmtstats` both have their respective `Collector` definitions, and `RemoteTopSQLReporter` implements both of them. If we renamed `tracecpu` to `collector/cpu`, then `stmtstats` should also be renamed to `collector/stmtstats`.